resources/views/account/partials/details/product/remove.blade.php

```diff
@@ -0,0 +1,7 @@
+@if($editable)
+    <td>
+        <button @click="removeItem(wishlist, item.entity_id)" class="hover:opacity-75 max-md:ml-auto">
+            <x-heroicon-s-heart class="w-5 h-5 text-ct-error" />
```

Comment:
should we introduce a new variable for the color of the wishlist?

Reply:
✅